var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -50,6 +52,16 @@ def patch(self):
         for file in glob.glob('*/CMakeLists.txt'):
             filter_file(r'.*/mod($|[^\w].*)', '', file)
 
+    def url_for_version(self, version):
+        # Version 4 uses a different URL syntax than previous versions
+        url_base = "https://github.com/Goddard-Fortran-Ecosystem/pFUnit"
+        if int(str(version.up_to(1))) >= 4:
```

Comment:
@adamjstewart Isn't there a simpler wayto get the first item in a version tuple?

Reply:
I would use:
```suggestion
        if version >= Version('4'):
```